src/components/notice/_notice.scss

```diff
@@ -0,0 +1,35 @@
+// Notice, text with an icon to the left
+.gv-c-notice {
+  @include clearfix;
+
+  position: relative;
+}
+
+.gv-c-notice__icon {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  margin-top: -17px; // Half the height of the important icon
+}
+
+.gv-c-notice__icon-fallback-text {
```

Comment:
For now, the visually-hidden styles are copied here.

We could use the utility class here: https://github.com/alphagov/govuk_frontend_alpha/blob/master/src/assets/scss/utilities/_visually-hidden.scss or, create a visually-hidden mixin.